.github/pull_request_template.md

```diff
@@ -0,0 +1,7 @@
+# Feature
+* Make a brief summary of the feature here should not be longer than two sentences
```

Comment:
minor typo, you need a comma or a period or smth here to make this grammatically correct